VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/data/Credentials.java

```diff
@@ -1,11 +1,26 @@
 package org.edx.mobile.test.feature.data;
 
+import android.support.annotation.Nullable;
+import org.edx.mobile.test.feature.config.E2EConfig;
+import org.edx.mobile.util.ResourceUtil;
+
+import java.util.UUID;
+
 public class Credentials {
     public final String email;
     public final String password;
+    @Nullable  public String username;
```

Comment:
Eventually we'll be registering the user for LoginFeatureTest (and the other tests) through the API instead of hard coding one (as we do on iOS) at which point we'll have both, but I guess making it nonnull can wait until then.